### ISSUE TITLE
TrustedCertEntry not being added to JKS together with PrivateKeyEntry

### ISSUE THREAD

**Issue by trunet** [kurtbrose/pyjks#37]
I'm generating a JKS from here:
```
def write_to_appendable_file(filename, data):
    filename.seek(0)
    filename.truncate()
    filename.write(data)

def save_to_jks(data):
    tce = jks.TrustedCertEntry.new('IssuingCA', data['issuing_ca'].encode())
    pke = jks.PrivateKeyEntry.new(args.domain, [data['certificate'].encode()], data['private_key'].encode(), 'rsa_raw')
    keystore = jks.KeyStore.new('jks', [pke, tce])
    keystore_bytes = keystore.saves(args.password)
    write_to_appendable_file(args.jks, keystore_bytes)
```

Running: keytool -list -v -keystore keystore.jks gives:
```
Keystore type: JKS
Keystore provider: SUN

Your keystore contains 2 entries

Alias name: test.mydomain.com
Creation date: Mar 7, 2018
Entry type: PrivateKeyEntry
Certificate chain length: 1
...
*******************************************
*******************************************


keytool error: java.lang.Exception: Alias <IssuingCA> does not exist
java.lang.Exception: Alias <IssuingCA> does not exist
	at sun.security.tools.keytool.Main.doPrintEntry(Main.java:1866)
	at sun.security.tools.keytool.Main.doPrintEntries(Main.java:2236)
	at sun.security.tools.keytool.Main.doCommands(Main.java:1123)
	at sun.security.tools.keytool.Main.run(Main.java:366)
	at sun.security.tools.keytool.Main.main(Main.java:359)
```

**Comment by djlambert** [comment]
I think this might be the same issue as #38. I was getting the same error when storing a ```TrustedCertEntry``` without  a ```PrivateKeyEntry ```, and when I lowercase the alias ```keytool -list``` works.

**Comment by magnuswatn** [comment]
Yes, this is a case issue. If you use:

`keytool -list -v -keystore keystore.jks -storetype CaseExactJKS`

it should work.

**Comment by mahmoud** [comment]
Seems like CaseExactJKS is the exception and not the rule. Version 18.0.0 of pyjks included a PR (#39) which automatically lowercases aliases to avoid this issue in future. Hoping this compromise works for everyone. Thanks all!
